canonical-websites/snapcraft.io
# Create publisher snaps blueprint

# Summary

Create publisher snaps blueprint

# QA

- `./run`
- Make sure publisher pages still work

```
publisher_snaps.get_account_snaps       GET        /account/snaps/
publisher_snaps.get_listing_snap        GET        /account/snaps/<snap_name>/listing
publisher_snaps.get_market_snap         GET        /account/snaps/<snap_name>/market
publisher_snaps.get_measure_snap        GET        /account/snaps/<snap_name>/measure
publisher_snaps.post_listing_snap       POST       /account/snaps/<snap_name>/listing
publisher_snaps.publisher_snap_metrics  GET        /account/snaps/<snap_name>/metrics
```
